package.json

```diff
@@ -12,13 +12,15 @@
         "@fortawesome/fontawesome-free": "^5.11.2",
         "copy-webpack-plugin": "^11.0.0",
         "dompurify": "^2.5.4",
+        "i18next": "^23.15.1",
```

Comment:
Please add i18next and react-i18next to the devDependencies. The GitHub Actions run the 'make dev' command, which is causing the test failure.